setup.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "optimade",
+    "version": "0.2.0",
+    "url": "https://github.com/Materials-Consortia/optimade-python-tools",
+    "license": "MIT",
+    "author": "OPTiMaDe Development Team",
+    "author_email": "dev@optimade.org",
+    "description": "Tools for implementing and consuming OPTiMaDe APIs.",
+    "keywords": "optimade jsonapi materials",
+    "include_package_data": true,
+    "package_data": {"": ["*.lark"]},
+    "classifiers": [
+        "Development Status :: 3 - Alpha",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Intended Audience :: Developers",
+        "Topic :: Database",
+        "Topic :: Database :: Database Engines/Servers",
+        "Topic :: Database :: Front-Ends"
+    ],
+    "python_requires": ">=3.7",
+    "install_requires": [
+        "pymongo>=3.8",
+        "lark-parser>=0.7.7",
+        "mongomock>=3.16",
+        "fastapi[all]>=0.42.0"
+    ],
+    "extras_require": {
+        "dev": ["black", "invoke", "pre-commit", "twine"],
+        "testing": ["pytest>=3.6", "pytest-cov", "openapi-spec-validator", "jsondiff"],
+        "django": ["django>=2.2.5"],
+        "elastic": ["elasticsearch_dsl>=6.4.0"]
```

Comment:
What's the strategy now for pinning deps to particular versions? Should the Pipfile be updated to include the pinned deps?

Reply:
As it is now, the Pipfile does nothing, I believe. I do not use `pipenv`, but I could look into how it works? In my opinion, for now, we simply keep the "minimum" versions put here by the Transformer class developers, and if at some point our tests fail, or they raise an issue with a newer version, I suggest we bug-fix and cap the dependency versions for that specific package?
I.e., this means we do not pin to specific versions, only minimum versions.

Reply:
Okay; happy with this provided version-related bug fixes are handled in separate PRs when they arise (and as long as we keep an eye on deprecation warnings as they crop up)

Reply:
Agreed! Great :+1: 
This is a sentiment that should be picked up and added to README and future documentation I think.